testing/oauth_server.ts

```diff
@@ -22,6 +25,67 @@ interface TokenCallbackResponse {
   data: {[key: string]: string};
 }
 
+async function requestOAuthAccessToken(
```

Comment:
MIght move this to an oauth_helpers.ts file or something, because the client credentials flow won't be using the oauth server.